includes/class-sensei-quiz.php

```diff
@@ -1904,6 +1905,12 @@ class="wp-block-button__link button quiz-submit complete sensei-course-theme__bu
 				</div>
 			<?php endif ?>
 
+			<?php if ( $next_lesson_url ) : ?>
+				<a class="wp-element-button sensei-course-theme__button is-primary" href="<?php echo esc_url( $next_lesson_url ); ?>">
+					<?php esc_attr_e( 'Continue to next lesson', 'sensei-lms' ); ?>
```

Comment:
Should this be `esc_html_e` since it's not an attribute?

Reply:
Thanks! Makes sense. Looks like all the other buttons in the footer action block were also using `esc_attr`, So I've updated them all here https://github.com/Automattic/sensei/pull/7202/commits/7d266b0ed1e2fd22a6804082c55af5ad3762e805